examples/cassandra/README.md

```diff
@@ -53,54 +53,48 @@ This is a somewhat long tutorial.  If you want to jump straight to the "do it no
 In Kubernetes, the atomic unit of an application is a [_Pod_](../../docs/user-guide/pods.md).  A Pod is one or more containers that _must_ be scheduled onto the same host.  All containers in a pod share a network namespace, and may optionally share mounted volumes.
 In this simple case, we define a single container running Cassandra for our pod:
 
-<!-- BEGIN MUNGE: EXAMPLE cassandra-controller.yaml -->
+<!-- BEGIN MUNGE: EXAMPLE cassandra.yaml -->
 
 ```yaml
 apiVersion: v1
-kind: ReplicationController
+kind: Pod
```

Comment:
This change is not appropriate. What are you trying to do?

https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/config-best-practices.md#naked-pods-vs-replication-controllers-and-jobs